compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -114,6 +114,7 @@ void quantize_and_verify(loco::Graph *g, Type quantized_dtype, Granularity granu
 
 // Helper function to reduce duplicate test codes
 // Assumption: g->output()->from() is the target node
+// TODO: Remove the function and use below function to explicit target node
```

Comment:
The previous helper functions assumed the target node as g->output->from() node, but now that we have both speficied-target test helper, we can replace the function to explicit the target node.

Reply:
@jinevening PTAL :-D

Reply:
IMHO, I think we can keep this function to avoid changing test cases.

Instead, we can implement this function with your function (with the `target`) to reduce duplicate codes.